### DISCUSSION TITLE
Custom framework scan

### DISCUSSION THREAD

**Discussion by hunter04** [kubescape/kubescape#516] [Q&A] [answered]
How can i run a scan to use only the custom framework i created?

**Comment by dwertent** [comment, marked as answer]
```
kubescape scan framework <custome framework name>
```

**Reply by hunter04** [reply]
Thanks @dwertent . 
When i try to run the command , i get this error.
[fatal] failed to download policies: 'Framework: test'. Make sure the policy exist and you spelled it correctly. For more information, please feel free to contact ARMO team

**Reply by dwertent** [reply]
Do you see the correct account ID when you run (don't share here your account ID)
```
kubescape config view
```
If not, make sure to run the scanning command with the `--account` flag.
You can find the account ID in the UI. LMK if you need help finding it.
If the account ID is the correct one, what do you see when you run:
```
kubescape list frameworks
```

**Reply by hunter04** [reply]
Thanks again @dwertent . That did the trick

**Reply by dwertent** [reply]
Sure, anytime :)
